src/app/contribute/guidelines/components/index.html

```diff
@@ -0,0 +1,240 @@
+      </li>
+    </ul>
+    <li>
+      <b>If you make an API change that affects inputs and outputs for a component, we recommend that you provide your potential (not working) documentation and demo to the SKY UX team for review.</b> This lets us head off API issues before implementation. Contributors should keep in mind that the API can change in the course of development.
+    </li>
+  </ul>
+
+  <stache-page-anchor>
+    Localization
+  </stache-page-anchor>
+
+  <p>
+    To declare localization strings, specify a string name and provide the string and a description in the <stache-code>resources_en_US.json</stache-code> file in <stache-code>src/assets/locales</stache-code>. You can reference the localization strings with the <stache-code>skyAppResources</stache-code> pipe or the <stache-code>SkyAppResourcesService</stache-code> service in <stache-code>@skyux/i18n</stache-code>.
+    </p>
+
+  <stache-page-anchor>
+    Get the code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      <a href="https://help.github.com/articles/fork-a-repo/">Fork the master branch</a> into your own GitHub repo.
+    </li>
+    <li>
+      Create a branch named after the feature you will contribute. The branch name should be kebab-case (my-new-feature).
+    </li>
+    <li>
+      Clone your repo locally and run <stache-code>npm install</stache-code> from your local repo's directory to install all required dependencies.
+    </li>
+    <li>
+      <a stacheRouterLink="." fragment="write-visual-regression-tests"></a>Run your first round of visual regression tests</a>. This establishes baseline screenshots of the components in the master repo so that as you make changes and run your visual regression tests, your changes are validated against the state of the repo before the changes.
+    </li>
+  </ol>
+
+  <stache-page-anchor>
+    Write code
+  </stache-page-anchor>
+
+  <ol>
+    <li>
+      Launch a command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>npm run watch</stache-code>.
+    </li>
+    <li>
+      Launch a second command prompt, <stache-code>cd</stache-code> to the folder where you cloned your branch, and run <stache-code>skyux serve</stache-code>. This launches the SKY UX component demos page where you write the demo components to test your code.
+    </li>
+    <ul>
+      <li>
+        SKY UX demos are authored with <a href="https://github.com/blackbaud/skyux-sdk-cli">SKY UX SDK CLI</a>. If this is your first time using SKY UX Builder, you must install the CLI as a global NPM package before you run <stache-code>skyux serve</stache-code>:
```

Comment:
Are we calling this "SKY UX SDK CLI"? I'd be okay with just "SKY UX CLI".

Reply:
Switched to SKY UX CLI